microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -837,7 +837,7 @@ def _install_ninja_and_meson(self) -> None:
             dest_dir=str(cwd),
             sudo=True,
         )
-        node.tools[Mv].move("ninja", "/usr/bin/ninja", overwrite=True, sudo=True)
+        node.tools[Mv].move(f"{cwd}/ninja", "/usr/bin/ninja", overwrite=True, sudo=True)
```

Comment:
cwd can be an argument of mv. It can be implied in src or dst.

Reply:
ok, will update